models/base/scratch/databricks/snowplow_media_player_base_events_this_run.sql

```diff
@@ -33,7 +33,7 @@ with prep AS (
       domain_sessionid,
       derived_tstamp
 
-      {% if var('snowplow__enable_load_tstamp', true) %}
+      {% if not var('snowplow__enable_load_tstamp', true) %}
```

Comment:
It's a typo, if you expand a few lines above this is within an exclude block - so this will match snowflake + bigquery so that's its actually correct.